lib/components/chat_history/twitch/subscription_event.dart

```diff
@@ -52,12 +56,13 @@ class TwitchSubscriptionGiftEventWidget extends StatelessWidget {
                 style: Theme.of(context).textTheme.titleSmall),
             TextSpan(
                 text:
-                    " gifted ${model.total} Tier ${model.tier.replaceAll("000", "")} "),
+                    " ${localizations!.gifted} ${model.total} ${localizations.tier} ${model.tier.replaceAll("000", "")} "),
             TextSpan(
-                text: model.total > 1 ? "subscriptions. " : "subscription. "),
```

Comment:
Do you know what this part of the logic was I don't remember off the top of my head, would like to make the translation part more descriptive for our translators.